cli/tests/unit/flash_test.ts

```diff
@@ -2164,8 +2059,9 @@ for (const [name, req] of badRequests) {
   );
 }
 
+// TODO(bartlomieju): fix this test
 Deno.test(
-  { permissions: { net: true } },
+  { permissions: { net: true }, ignore: true },
```

Comment:
@littledivy please take a look

Reply:
I don't understand why this test requires `Content-Length: 0` for the HEAD requests. It appears Hyper is not doing that and it's not mandatory to send this header. Can we remove this test case?

Reply:
There shouldn't be any need for a zero `Content-Length` header, implicit or explicit. Does `node` have one? Even then, that's probably better to keep in the polyfill.